src/shared/components/guidedTour/useStepDispatch.hook.tsx

```diff
@@ -0,0 +1,11 @@
+import { useDispatch } from 'react-redux';
+
```

Comment:
There is a weird bug in this library, causing it to break when `redux` is used. By using `setTimeout` with `0ms` we are pushing dispatch further in the call stack, which results in the library working properly and not causing a delay